docs/sphinx/omero-pyramid/index.rst

```diff
@@ -0,0 +1,27 @@
+The OMERO pyramid format
+========================
+
+The OMERO pyramid format is a way of storing very large images for easier visualization.
+Currently only v1.0.0 is currently defined.
```

Comment:
Just one little nitpick, you've got currently twice in the this sentence, you only need the first one.

Reply:
Should be fixed with ee55320